envoy.dependency.check/envoy/dependency/check/abstract/checker.py

```diff
@@ -21,7 +26,15 @@ class ADependencyChecker(
         metaclass=abstracts.Abstraction):
     """Dependency checker."""
 
-    checks = ("cves", )
+    checks = ("cves", "dates")
```

Comment:
should this have a better name than simply "dates" ?

Reply:
yep, i think that would be better

how about `release-dates` as then its consistent with `releases` and `release-issues`

Reply:
yeah that would be good